README.md

```diff
@@ -61,6 +61,36 @@ To execute tests, run
 
 Refer to [SBT docs](https://www.scala-sbt.org/1.x/docs/Command-Line-Reference.html) for more commands.
 
+# IntelliJ Setup
+
+IntelliJ is the recommended IDE to use when developing Delta Lake. To import Delta Lake as a new project:
+1. Clone Delta Lake into, for example, `~/delta`.
+2. In IntelliJ, select `File` > `New Project` > `Project from Existing Sources...` and select `~/delta`.
+3. Under `Import project from external model` select `sbt`. Click `Next`.
+4. Under `Project JDK` specify a valid Java `1.8` JDK and opt to use SBT shell for `project relaod` and `builds`.
+5. Click `Finish`.
+
+## Setup Verification
+
+After waiting for IntelliJ to index, verify your setup by running a test suite in IntelliJ.
+1. Search for and open `DeltaLogSuite`
+2. Next to the class declaration, right click on the two green arrows and select `Run 'DeltaLogSuite'` 
+
+## Troubleshooting
+
+If you see errors of the form
+
+```
+object DeltaSqlBaseParser is not a member of package io.delta.sql.parser
```

Comment:
Want to include the "error" part of the message? Just looks a little weird as an error like this.